src/main/java/com/nazar/carstudio/controller/NazarController.java

```diff
@@ -1,57 +1,34 @@
 package com.nazar.carstudio.controller;
 
+import com.nazar.carstudio.service.NazarService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.HashMap;
-import java.util.Map;
-
 @RestController
 public class NazarController {
 
+    @Autowired
+    private final NazarService nazarService;
+
+    public NazarController(NazarService nazarService) {
+        this.nazarService = nazarService;
+    }
```

Comment:
if you use @Autowired, you don't need constructor. Use 
@Autowired
    private NazarService nazarService;